pandas/io/formats/latex.py

```diff
@@ -384,8 +388,23 @@ def _position_macro(self) -> str:
 
     @property
     def _caption_macro(self) -> str:
-        r"""Caption macro, extracted from self.caption, like \caption{cap}."""
-        return f"\\caption{{{self.caption}}}" if self.caption else ""
+        r"""Caption macro, extracted from self.caption.
+
+        With short caption:
+            \caption[short_caption]{caption_string}.
+
+        Without short caption:
+            \caption{caption_string}.
+        """
+        if self.caption:
+            return f"\\caption{self._short_caption_macro}{{{self.caption}}}"
```

Comment:
```suggestion
            return f"\\caption{self._short_caption or ''}{{{self.caption}}}"
```

This can be simplified to get rid of the property

Reply:
``short_caption_macro`` is either an empty string or ``[short caption text]`` (in square brackets).
Putting this logic into f-string seems to be rather complicated.
IMHO having a separate property (``_short_caption_macro``) with clear intent is more readable.

Reply:
@WillAyd, does my answer make sense?
@jreback, @simonjayhawkins, @toobaz, any plans on merging this?
Or does it require additional work?

Reply:
@ivanovmg am looking now, we have 200 open PRs; these take time

Reply:
the way you have it is ok; the f-string actually is pretty complicated to grok on the substitutions. if you can make it simpler would take (e.g. use concatenation maybe)

Reply:
I made string joining instead of long f-string. Would you consider it more readable?